src/System.Buffers.Experimental/System/Buffers/BufferExtensions.cs

```diff
@@ -325,21 +324,11 @@ internal static int IndexOfStraddling(this ReadOnlySpan<byte> first, IReadOnlyBu
             // this check is a small optimization: if the first byte from the value does not exist in the bytesToSearchAgain, there is no reason to combine
             if (bytesToSearchAgain.IndexOf(value[0]) != -1)
             {
-                Span<byte> combined;
                 var combinedBufferLength = value.Length << 1;
-                if (combinedBufferLength < 128)
-                {
-                    unsafe
-                    {
-                        byte* temp = stackalloc byte[combinedBufferLength];
-                        combined = new Span<byte>(temp, combinedBufferLength);
-                    }
-                }
-                else
-                {
-                    // TODO (pri 3): I think this could be eliminated by chunking values
-                    combined = new byte[combinedBufferLength];
-                }
+                Span<byte> combined = combinedBufferLength < 128 ?
+                                        stackalloc byte[combinedBufferLength] :
+                                        // TODO (pri 3): I think this could be eliminated by chunking values
+                                        combined = new byte[combinedBufferLength];
```

Comment:
Is this "combined =" on purpose or a mistake?

Reply:
typo, it is a redundant assignment